Source/PageView/PageView.swift

```diff
@@ -24,7 +24,7 @@ class PageView: UIView {
   }
   
   fileprivate var containerX: NSLayoutConstraint?
-  fileprivate var containerView: PageContrainer?
+  var containerView: PageContrainer?
```

Comment:
Please explain the change here.

Reply:
This change make a variable from private to be public inside the component, because it needs to be visited in 
`let items = containerView.items`
marked below.